docs/canonicalk8s/snap/reference/upgrading.md

```diff
@@ -0,0 +1,55 @@
+# Upgrade notes
```

Comment:
I think we should possibly add a warning that you should change to the release-1.33 version of the docs here on RTD so you get the most updated docs